src/raspbian/10/helix/arm32v6/Dockerfile

```diff
@@ -1,13 +1,21 @@
-# dummy F R O M to establish a dependency
-FROM mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-10-crossdeps-local
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-10-crossdeps-local as builder
+
+ARG ROOTFS_DIR=/crossrootfs
+
+# Obtain arcade scripts used to build rootfs
+RUN git config --global user.email builder@dotnet-buildtools-prereqs-docker && \
+    git clone --depth 1 --single-branch https://github.com/dotnet/arcade /scripts
```

Comment:
It doesn't feel right that we are cloning arcade tip.  We really should be using the version of arcade this repo has.  Tip arcade hasn't been validated through it's inner validation process.

Reply:
> We really should be using the version of arcade this repo has.

There is no Arcade dependency flow to this repo.

Reply:
It's also no different than what was previously being done in run-arcade-build-rootfs.sh. So that should be addressed as separate work if it is deemed an issue.

Reply:
I see we are currently doing this in the run-arcade-build-rootfs.sh script.  I thought we utilizing dependency flow for isolation.  I guess that is a separate issue - it obviously has not been a source of problems.  You can ignore my comment.